common/src/main/java/com/wynntils/features/user/InfoMessageFilterFeature.java

```diff
@@ -176,4 +193,21 @@ public void onInfoMessage(ChatMessageReceivedEvent e) {
             }
         }
     }
+
+    private static void sendLoginMessage(String playerName, String rank) {
+        switch (rank) {
+            case "VIP" -> NotificationManager.queueMessage(
+                    ChatFormatting.GREEN + "→ " + ChatFormatting.DARK_GREEN + "[" + ChatFormatting.GREEN + "VIP"
+                            + ChatFormatting.DARK_GREEN + "] " + ChatFormatting.GREEN + playerName);
+            case "VIP+" -> NotificationManager.queueMessage(
+                    ChatFormatting.GREEN + "→ " + ChatFormatting.DARK_AQUA + "[" + ChatFormatting.AQUA + "VIP+"
+                            + ChatFormatting.DARK_AQUA + "] " + ChatFormatting.AQUA + playerName);
+            case "HERO" -> NotificationManager.queueMessage(
+                    ChatFormatting.GREEN + "→ " + ChatFormatting.DARK_PURPLE + "[" + ChatFormatting.LIGHT_PURPLE
+                            + "HERO" + ChatFormatting.DARK_PURPLE + "] " + ChatFormatting.LIGHT_PURPLE + playerName);
+            case "CHAMPION" -> NotificationManager.queueMessage(
+                    ChatFormatting.GREEN + "→ " + ChatFormatting.YELLOW + "[" + ChatFormatting.GOLD + "CHAMPION"
+                            + ChatFormatting.YELLOW + "] " + ChatFormatting.GOLD + playerName);
```

Comment:
This lacks a default branch, and has a decent bit of duplicated code

Reply:
No need for default branch, I will fix duplication.

Reply:
We thought so too when HERO was the highest rank

Reply:
Champ branch does not even work, wynn sends hero format for them. Also, we can't just put an universal format.